modules/swagger-codegen/src/test/scala/Objc/ObjcModelTest.scala

```diff
@@ -173,7 +170,7 @@ class ObjcModelTest extends FlatSpec with Matchers {
     val vars = cm.vars
     vars.get(0).baseName should be ("children")
     vars.get(0).complexType should be ("SWGChildren")
-    vars.get(0).datatype should be ("NSArray*")
+    vars.get(0).datatype should be ("NSArray<SWGChildren>*")
```

Comment:
If the item of array is a model, the declaration of model property data type should be `NSArray<SWGChildren>*`.
JSONModel use the protocal `SWGChildren` to determine the inner type of the array of the property.